public/theme/boltcms-2021/listing.twig

```diff
@@ -0,0 +1,30 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main_container %}
+
+    <div class="content-wrapper">
+        <div class="current-version">
+            <p>Latest version of Bolt CMS: 4.1.13 <a href="https://github.com/bolt/core/releases/latest">Download this
+                    version</a></p>
+        </div>
+        <div class="important-message">
+            <div class="header-block">
+                <h2>Bolt 4.1.13 released</h2>
+                <span transition-style="in:circle:top-left">#Announcement</span>
+            </div>
+            <p>Bolt 4.1.13 brings a slew of accessibility, usability, and DX improvements.</p>
+        </div>
+        {% setcontent record = 'news' limit 3 %}
```

Comment:
Here, if we limit the results to 3 items, visitors won't be able to see more than the 3 most recent news items.

By moving this thole code from `listing.twig` to `news.twig`, we'll automatically get a variable `records` which will have the correct number of items. Also, it will be paginated, meaning that we'll be able to see page 2, 3, 4, etc. :-)

As far as I know, there's no pager design yet. Looks like a good time to ask Derk for it!

Reply:
We have to take a look at this...

Reply:
Done